test/test_jit.py

```diff
@@ -1061,7 +1061,34 @@ def observe(x, name):
         self.assertEqual(value_stats['z'][1], x2 - y2)
 
     def test_expand_insert_fakequant(self):
-        pass
+        class testModule(torch.jit.ScriptModule):
+            def __init__(self):
+                super(testModule, self).__init__()
+                self.conv1 = nn.Conv2d(1, 20, 5, 1)
+                self.conv2 = nn.Conv2d(20, 50, 5, 1)
+                self.fc1 = nn.Linear(4 * 4 * 50, 500)
+                self.fc2 = nn.Linear(500, 10)
+
+            @torch.jit.script_method
+            def forward(self, x):
+                x = F.relu(self.conv1(x))
+                x = F.max_pool2d(x, 2, 2)
+                x = F.relu(self.conv2(x))
+                x = F.max_pool2d(x, 2, 2)
+                x = x.view(-1, 4 * 4 * 50)
+                x = F.relu(self.fc1(x))
+                x = self.fc2(x)
+                return x
+
+        trace = testModule()
+        torch._C._jit_pass_constant_propagation(trace.graph)
```

Comment:
Please add a comment explaining why we execute const-prop here.

Reply:
Can we elaborate more? What is the constant prop pass doing? If I only need to know the op (eg relu) whose output I need to quantize, why do I need this pass? Can you clarify in the comments?